app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-core.js

```diff
@@ -288,9 +288,10 @@ window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {};
           return null
         }
 
-        // In order to extract the number of results from resultCount (which is a string at this point (e.g. '12,345 results')), we remove the comma and
+        // In order to extract the number of results from resultCount (which is a string at this point (e.g. '12,345 results' or 'Results: 1,234 licenses')), we remove 'Results: ' and the comma, then
         // split string at the space character so it can be parsed as an integer
-        resultCount = window.GOVUK.analyticsGa4.core.trackFunctions.removeLinesAndExtraSpaces(resultCount.textContent)
+        resultCount = resultCount.textContent.replace('Results: ', '')
```

Comment:
Confused about how this matches both `Results: xx` and `xx results`, can you explain? Presumably that's handled elsewhere?

Reply:
Would it be more flexible to match the number using a regex? (e.g. `[0-9,]+`) - this would account for both variations.